### DISCUSSION TITLE
About filament runout settings

### DISCUSSION THREAD

**Discussion by DonRonito** [mriscoc/Ender3V2S1#279] [Q&A] [answered]
Have been experimenting for a while with the runout settings (and have read others wondering the same). Instructions say «Selectable runout filament sensor active state (LOW/HIGH/MOTION)». What are these different states? Do they refer to different type of sensors, or do they have different funtionality? Will any one of these stop the print and wait for filament refill before then continuing the print?

**Comment by mriscoc** [comment, marked as answer]
https://github.com/mriscoc/Ender3V2S1/wiki/Filament-Settings-Menu

**Reply by DonRonito** [reply]
Thanks a lot. Did not find this before and it explains the behaviour I was wondering about. Is it possible to load more filament and the printer continues (after the runout distance is reached and prints stop), or is that a feature not simply supported by Marlin?

**Reply by mriscoc** [reply]
Maybe in `Load mm` setting in that menu.
